### PR TITLE
Add GetAxisPar (including "shape")

### PR DESCRIPTION
**Pull request by tiagocoutinho**
Add GetAxisPar

Includes "shape" according to new sardana requirements.

### CODE REVIEW

File: sardana_limaccd/ctrl/LimaCCDCtrl.py [hunk 1]
```diff
@@ -473,6 +473,16 @@ def GetCtrlPar(self, parameter):
...
 
+    def GetAxisPar(self, axis, parameter):
+        if parameter == "value_ref_pattern":
+            return self._value_ref_pattern
+        elif parameter == "value_ref_enabled":
+            return self._value_ref_enabled
+        elif parameter == "shape":
+            data = self._limaccd.read_attributes(("image_width", "image_height"))
+            return data[0].value, data[1].value
+        return super().GetAxisPar(axis, parameter)
```
Comment (jordiandreu) [inline sardana_limaccd/ctrl/LimaCCDCtrl.py]: This is not python2 compatible. Could we use this instead?:
`return super(LimaCCDTwoDController, self).GetAxisPar(axis, parameter)`

**Review by jordiandreu — CHANGES_REQUESTED** [review]

File: sardana_limaccd/ctrl/LimaCCDCtrl.py (hunk 1 above)
Comment (tiagocoutinho) [reply sardana_limaccd/ctrl/LimaCCDCtrl.py]: Yes, sure! Thanks for pointing it out.

**Review by tiagocoutinho — COMMENTED** [review]
Ok, it should be fixed now

**Comment by jordiandreu** [comment]
Great! @tiagocoutinho thanks for the change, now is fine for me (already tested in python2).

**Review by jordiandreu — APPROVED** [review]

**Review by rhomspuron — APPROVED** [review]

**Pull request merged**